src/rich-log/domain.hh

```diff
@@ -14,49 +18,62 @@ namespace verbosity
 ///   - everything else is enabled
 enum type : int
 {
+    _None = 0,
```

Comment:
I don't see the benefit of having a hole here. Where would we use that? And if we really need it, why not at the end? (This design makes it harder to bucket by verbosity for gain that I don't see yet)

Reply:
Hm i didn't consider the "hole" for bucketing, that's a good point. The motivation was to have an invalid value in the first place (and zero being invalid is just nice for zero-init and so forth)